docs/guide-ssl-setup.md

```diff
@@ -57,3 +57,8 @@ ssl/client_ciphersuites = 150:PFS:HIGH::EC_P256:EC_HIGH
 ssl/ciphersuites = 135:PFS:HIGH::EC_P256:EC_HIGH
 icm/HTTPS/client_sni_enabled = TRUE
 ```
+
+- for NPL minisap - add profile parameters to following file:
```

Comment:
Maybe add the official title is "SAP NetWeaver AS ABAP Developer Edition on ASE"

Note: "SAP NetWeaver AS ABAP Developer Edition on SAP HANA" has a different sysid/path. I think it's /usr/sap/A4H/sys/profiles. 

https://developers.sap.com/trials-downloads.html?search=abap